config/samples/backends/hpe/3par/fc/cinder-volume-3par-secrets.yaml

```diff
@@ -0,0 +1,22 @@
+# Define the "cinder-volume-3par-secrets" Secret that contains sensitive
+# information pertaining to the [hpe3par] backend.
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    service: cinder
+    component: cinder-volume
+  name: cinder-volume-3par-secrets
+type: Opaque
+stringData:
+  hpe3par-secrets.conf: |
```

Comment:
Were you going to sanitize the secrets files? The presumption is every entry (L14..L22) by definition needs to be filled in by the user.

Reply:
Yes, they should be filled in by the user, but I'm using the samples from the vendor's documentation.